### ISSUE TITLE
Feat: Add ESLint and Prettier Configuration

### ISSUE THREAD

**Issue by abhayymishraa** [AOSSIE-Org/PictoPy#69]
### Describe the feature

Currently our codebase lacks consistent code formatting and linting standards. This feature request is to add ESLint and Prettier configuration to maintain code quality and consistency.

Changes to be made:
- Add ESLint for code linting
- Add Prettier for consistent code formatting
- Configure necessary dependencies and rules
- Add scripts in package.json for linting and formatting

Expected Outcome:
- Consistent code formatting across the project
- Automated linting to catch potential issues
- Better code quality and maintainability

Actually it should be added in the code so that the code formatting is good in the overall repository also eslint will catch the potential error . For first thing is apply the low level then the changes would be made in the eslintrc.json file according to the need 

### Add ScreenShots

![image](https://github.com/user-attachments/assets/92f0943b-03bf-45b5-9321-68bcda70817c)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I want to work on this issue

**Comment by abhayymishraa** [comment]
I am working on the issue.

**Comment by abhayymishraa** [comment]
@Pranav0-0Aggarwal Sir i have added linting the project the linting is not strict . also sir if we need to add strict in the linting in order to do that we should be make a lot of changes i am ready to take this job tooo but for now no strict just i introduced linting also i initialized the project with the prettier with tailwind config so that it will work with each other

**Comment by abhayymishraa** [comment]
Also sir this many file changes is becuase of prettier format sir if u want i will remove the changes and will raise a pr with the functionality only also sir keep me updated about the eslint config thingy i will work on it and make it strict but sir also it require a lot of changes like use of "any "  will give error so as you say i will do it sir

**Comment by abhayymishraa** [comment]
This is the command  npm run format:fix
![Screenshot from 2024-11-19 00-38-38](https://github.com/user-attachments/assets/ebc54f0e-188a-4802-9d7c-f5601df94130)
This is the command npm run format:check
Also this is how it look after formatting
![Screenshot from 2024-11-19 00-37-51](https://github.com/user-attachments/assets/83808a36-7c31-4fdc-8bb6-902f909e0128)
more will  be added

**Comment by Pranav0-0Aggarwal** [comment]
is this issue resolved?

**Comment by abhayymishraa** [comment]
yep we can add strict rule in linitng if we want but that needs a lot of change like in many component i saw that we used (any) which is not good for some scenarios we should declare types . @Pranav0-0Aggarwal  if you  will assign the next things i will work on it and define feature. where needed

**Comment by Pranav0-0Aggarwal** [comment]
Assigned to you

**Comment by abhayymishraa** [comment]
@Pranav0-0Aggarwal  im working on it
